Codeinwp/neve
# fix: allow gutenberg styles to load on other gutenberg admin pages #2954

### Summary
Changed the condition for Gutenberg styles enqueue to allow other pages that support the Gutenberg to load the required styles.

### Will affect visual aspect of the product
YES

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/125267784-cb508b00-e30f-11eb-9344-a25cebc5a794.png)


### Test instructions
1. Install the latest beta version of wordpress
2. In Dashboard > Appearance > Widgets add a block to the sidebar
3. Select a palette color for the block
4. Check how it looks in editor/frontend


Closes #2954.
